src/service-worker/src/armada/cid.ts

```diff
@@ -0,0 +1,65 @@
+/**
```

Comment:
how heavy is it to add the proper module or even vendor the file vs reimplementing